features/networking/service.feature

```diff
@@ -532,3 +531,60 @@ Feature: Service related networking scenarios
       | type          | merge                                            |
     Then the step should fail
     And the output should contain "does not completely cover the previous range"
+
+  # @author hongli@redhat.com
```

Comment:
please update the author to you as well, that's helpful when someone checking log and contacting the owner directly.

Reply:
sure if you'd like. updated it